### DISCUSSION TITLE
Feature Request: Allow Session Recording on Remote Side

### DISCUSSION THREAD

**Discussion by Marc-Harony** [rustdesk/rustdesk-server-pro#437] [Q&A] [answered]
Hi ! :wave:

Currently, RustDesk allows session recording, but the recorded file is saved only on the initiator's side. For security and compliance purposes, it would be beneficial to enable session recording on the remote side as well. This would allow remote users to maintain their own records of sessions for audit trails and adherence to organizational policies.

**Proposed Solution:**

- Implement an option for remote users to record sessions on their own devices.
- Allow configuration of recording settings, such as file format and storage location.

**Benefits:**

- Enhances security by providing both parties with session records.
- Assists in compliance with organizational policies and regulatory requirements.
- Increases transparency and trust between users.

**Comment by rustdesk** [comment, marked as answer]
https://rustdesk.com/docs/en/self-host/client-configuration/advanced-settings/#allow-auto-record-incoming
